case_studies/README.md

```diff
@@ -1,7 +1,25 @@
 # Case Studies
 
-This directory contains case studies of events from the [Labs](../meta/labs)
-clubs and any others that choose to participate.
+This directory contains case studies of events from the
+[Labs](../meta/labs) clubs and any others that choose to participate.
+
+A case study is written for each of a club's events. Each case study is a story
+that paints a up-close, in-depth, and context-rich picture of all the planning
+and the aftermath of a given event.
+
+We want to document any and every event related to a club including things like:
+
+- the club fair
+- the first club meeting
+- club elections
+- pre-club planning meetings
+- any field trip that a club might organize
+- etc.
```

Comment:
The `etc` seems redundant because on line 10 you already said that the list is just a limited subset of a much larger set.